torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -550,11 +556,17 @@ def pad_image_tensor(
     num_channels, height, width = img.shape[-3:]
     extra_dims = img.shape[:-3]
 
-    padded_image = _FT.pad(
-        img=img.view(-1, num_channels, height, width), padding=padding, fill=fill, padding_mode=padding_mode
-    )
+    left, right, top, bottom = _FT._parse_pad_padding(padding)
```

Comment:
Nit: this part of code can be put into `else` part to avoid parsing twice for normal case